org.lflang.lfc/src/org/lflang/lfc/Main.java

```diff
@@ -111,7 +111,10 @@
         THREADS("t", "threads", true, false, "Specify the default number of threads.", true),
         OUTPUT_PATH("o", "output-path", true, false, "Specify the root output directory.", false),
         RUNTIME_VERSION(null, "runtime-version", true, false, "Specify the version of the runtime library used for compiling LF programs.", true),
-        EXTERNAL_RUNTIME_PATH(null, "external-runtime-path", true, false, "Specify an external runtime library to be used by the compiled binary.", true);
+        EXTERNAL_RUNTIME_PATH(null, "external-runtime-path", true, false, "Specify an external runtime library to be used by the compiled binary.", true),
+        RTI_HOST(null, "host", true, false, "Specify the hostname or IP address of the RTI.", true),
```

Comment:
I think using a single argument called `--rti` is more descriptive than having separate `host`, `port`, and `user` arguments without a context of what they are for.

Reply:
I'll add prefix `rti` (ex. `--rti_host`) for now.

Reply:
I don't personally think that `--rti_host` looks intuitive enough. I really think that we should aspire to come up with an elegant permanent solution rather than coming up with temporary fixes. If you don't want to tackle this, I can take the implementation of the `--rti` cli argument as an action item.